### PR TITLE
Clarify differences between AEMaaCS and AEM 6.5 for embedding

### PR DESCRIPTION
**Pull request by kwin**
List AEMaaCS first, still leverage subpackage for AEM6.5

### CODE REVIEW

File: acs-aem-commons/pages/maven.md [hunk 1]
```diff
@@ -14,73 +14,126 @@ Please refer to [Compatibility](/acs-aem-commons/pages/compatibility.html) to ch
...
-        <artifactId>filevault-package-maven-plugin</artifactId>
+  <plugin>
+    <groupId>org.apache.jackrabbit</groupId>
+    <artifactId>filevault-package-maven-plugin</artifactId>
+      ...
+      <configuration>
+        <embeddeds>
+          <embedded>
+            <!-- the artifact with the given id from the project dependencies is embedded -->
+            <artifactId>acs-aem-commons-all</artifactId>
```
Comment (DasAnk) [inline acs-aem-commons/pages/maven.md:42]: @kwin Don't we need <groupId>?

Comment (kwin) [reply acs-aem-commons/pages/maven.md:42]: No, this acts as filter on the project dependencies.

Comment (DasAnk) [reply acs-aem-commons/pages/maven.md:42]: @kwin I see the bundles also getting installed with the dependency defined in all's pom.xml. Do we need the core dependencies then?

image

Comment (kwin) [reply acs-aem-commons/pages/maven.md:42]: all is a container comprising everything relevant.

Comment (DasAnk) [reply acs-aem-commons/pages/maven.md:42]: @kwin Then we need not have the optional dependency in core/pom.xml as all/pom.xml will always deploy the bundles.

Comment (kwin) [reply acs-aem-commons/pages/maven.md:42]: Core != All. The former is an OSGi bundle which needs the ACS AEM Commons bundle as dependency to appear in the compile time class path in order to use ACS Commons API.

Comment (DasAnk) [reply acs-aem-commons/pages/maven.md:42]: Thanks for clarifying @kwin. I was only thinking runtime since I noticed the bundles installed. Thanks again!

**Pull request merged**